patches/0001-sys-newlib_syscalls_default-set-_init-and-_fini-as-w.patch

```diff
@@ -0,0 +1,75 @@
+From 945e5c94e727969e1f2964f27ca7e0e4ec7ee695 Mon Sep 17 00:00:00 2001
+From: Gilles DOFFE <g.doffe@gmail.com>
+Date: Mon, 1 Aug 2022 15:35:24 +0200
+Subject: [PATCH] sys/newlib_syscalls_default: set _init() and _fini() as weak
+
+The _init() and _fini() functions are already defined into the
+C RunTime (crti.o) object file provided by the arm-none-eabi toolchain.
+To avoid multiple definition error, set them as 'weak'.
+
+Signed-off-by: Gilles DOFFE <g.doffe@gmail.com>
+---
+
+From 945e5c94e727969e1f2964f27ca7e0e4ec7ee695 Mon Sep 17 00:00:00 2001
+From: Gilles DOFFE <g.doffe@gmail.com>
+Date: Mon, 1 Aug 2022 15:35:24 +0200
+Subject: [PATCH] sys/newlib_syscalls_default: set _init() and _fini() as weak
+
+The _init() and _fini() functions are already defined into the
+C RunTime (crti.o) object file provided by the arm-none-eabi toolchain.
+To avoid multiple definition error, set them as 'weak'.
+
+Signed-off-by: Gilles DOFFE <g.doffe@gmail.com>
+---
+
+From 945e5c94e727969e1f2964f27ca7e0e4ec7ee695 Mon Sep 17 00:00:00 2001
+From: Gilles DOFFE <g.doffe@gmail.com>
+Date: Mon, 1 Aug 2022 15:35:24 +0200
+Subject: [PATCH] sys/newlib_syscalls_default: set _init() and _fini() as weak
+
+The _init() and _fini() functions are already defined into the
+C RunTime (crti.o) object file provided by the arm-none-eabi toolchain.
+To avoid multiple definition error, set them as 'weak'.
+
+Signed-off-by: Gilles DOFFE <g.doffe@gmail.com>
+---
+
+From 945e5c94e727969e1f2964f27ca7e0e4ec7ee695 Mon Sep 17 00:00:00 2001
+From: Gilles DOFFE <g.doffe@gmail.com>
+Date: Mon, 1 Aug 2022 15:35:24 +0200
+Subject: [PATCH] sys/newlib_syscalls_default: set _init() and _fini() as weak
+
+The _init() and _fini() functions are already defined into the
+C RunTime (crti.o) object file provided by the arm-none-eabi toolchain.
+To avoid multiple definition error, set them as 'weak'.
+
+Signed-off-by: Gilles DOFFE <g.doffe@gmail.com>
+---
```

Comment:
The beginning of the patch is weird, the header appears 4 times...

Reply:
:fearful:

Reply:
Dunno why :open_mouth: but fixed